ynput/ayon-frontend
# Enhancement: Editor Selection Refinement

### Description

- Fix: Issue where selected tasks in `Hierarchy` would be stuck in `Editor`.
- Chore: Created new `focused.editor` state for better selection management between pages.
- Feature: Selecting folders in `Hierarchy` will auto expand the folders in `Editor`. If any tasks are selected these will also be auto selected.

### Testing

1. Go to `Hierarchy` page and select a folder and then a task.
2. Go to `Editor` page and that folder should be expanded and the task selected.

### Issues

- When lots of folders have been selected in `Hierarchy`; `Editor` can take a very long time to load all expanded folders.
